protyping/wlutz/wlutz-with-pylinac.ipynb

```diff
@@ -0,0 +1,160 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import pathlib"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pymedphys import _losslessjpeg as lljpeg\n",
+    "from pymedphys._wlutz import iview as _iview\n",
+    "from pymedphys._wlutz import imginterp as _imginterp\n",
+    "from pymedphys._wlutz import findfield as _findfield\n",
+    "from pymedphys._wlutz import findbb as _findbb\n",
+    "from pymedphys._wlutz import reporting as _reporting"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def read_image(path):\n",
+    "    return lljpeg.imread(path)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "path = pathlib.Path(\n",
+    "    r'\\\\tunnel-nbcc-pdc\\Physics\\NBCC-DataExchange\\iViewDB\\patient_MVISO\\MV_IMAGES\\img_1.3.46.423632.301843.1579156836.86\\00022.1.3.46.423632.301843.1579156836.86.jpg'\n",
```

Comment:
hardcoded path.
where is this going to work correctly, and where is it going to fail?
This looks like it's intended to be an example (or a unit test).
suggest extracting the "works only in my house" part so that it can be readily understood for what it is and modified readily by someone who needs it to work in their house.

Reply:
This one's a prototyping notebook. (in the prototyping directory). It's purely a "I'm trying to learn something for myself". So anything goes here (as long as the information isn't sensitive)